MapboxCoreNavigation/RouteController.swift

```diff
@@ -277,9 +290,11 @@ open class RouteController: NSObject {
         self.locationManager = locationManager
         self.locationManager.activityType = route.routeOptions.activityType
         self.eventsManager = eventsManager
-        UIDevice.current.isBatteryMonitoringEnabled = true
 
         super.init()
+        
+        let monitorBatteryValue = delegate?.routeController?(self, willChangeBatteryMonitoringState: true) ?? true
```

Comment:
`delegate` is always nil here.

Reply:
Moved in https://github.com/mapbox/mapbox-navigation-ios/pull/1476/commits/573d582c1be10a9f7e8a296f96147229033bcd5c